daq_macros.py

```diff
@@ -26,7 +26,10 @@
 from collections import OrderedDict
 from scans import (zebra_daq_prep, setup_zebra_vector_scan,
```

Comment:
Where is scans being imported from now that the old file is deleted?

Reply:
this should be coming from mxtools as with zebra below